website/pages/en/network/indexing.mdx

```diff
@@ -2,7 +2,7 @@
 title: Indexing
 ---
 
-Indexers are node operators in The Graph Network that stake Graph Tokens (GRT) in order to provide indexing and query processing services. Indexers earn query fees and indexing rewards for their services. They also earn from a Rebate Pool that is shared with all network contributors proportional to their work, following the Cobb-Douglas Rebate Function.
+Indexers are node operators in The Graph Network that stake Graph Tokens (GRT) in order to provide indexing and query processing services. Indexers earn query fees and indexing rewards for their services. They also earn query fees that are rebated according to an exponential rebate function.
```

Comment:
I think it might be worth providing a bit more of a legible explanation of what the rebate function is for (essentially ensuring that indexers have sufficient stake allocated when claiming a given amount of query fees)

Reply:
Agreed. To be explicit, after "to an exponential rebate function.", you could add:

> This aim of using this rebate function is to ensure indexers achieve the best outcome by faithfully serving queries. This works by incentivizing Indexers to allocate a large amount of stake (which can be slashed for erring when serving a query) relative to the amount of query fees they may collect.